dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/DataRequestManager.kt

```diff
@@ -298,11 +299,11 @@ class DataRequestManager(
             bulkDataRequest,
             acceptedCompanyIdentifiers,
         )
-        val bulkDataRequestNotificationMailLoggerFunction = {
-            dataRequestLogger
-                .logMessageForBulkDataRequestNotificationMail(emailToSend, bulkDataRequestId)
-        }
-        emailSender.sendEmail(emailToSend, bulkDataRequestNotificationMailLoggerFunction)
+        emailSender.sendEmail(
```

Comment:
Because the logging is now part of the .send function?

Reply:
Yes. But I'm not sure if we should keep the second parameter here. We could log this custom string also just before the call to .sendEmail

Reply:
Not a strong opinion, from my point of view

Reply:
I'll change it then. It feels less weird to me that way :D

Reply:
done